packages/cadl-rlc-emitter/test/integration/generated/models/inheritance/src/outputModels.ts

```diff
@@ -3,6 +3,16 @@ export interface SiameseOutput extends CatOutput {
   smart: boolean;
 }
 
+/** The second level model in the normal multiple levels inheritance. */
+export interface CatOutput extends PetOutput {
```

Comment:
Could you add the cadl-ranch test cases for this one?

Reply:
updated